pattern/ecs-fargate-container-image-signing-and-verification/files/sigverify/sigverify.py

```diff
@@ -0,0 +1,59 @@
+# // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# // SPDX-License-Identifier: Apache-2.0
+import json
+import subprocess
+import boto3
+import base64
+import os
+from datetime import datetime
+
+# ------ static initialization code ------
+region=os.environ["AWS_REGION"]
+
+# Notation normally requires a credentials store and crendentials store helper, but
+# in Lambda we are not able to use these.
+def notation_verify_signature(ecr_client, container_image):
+    auth_token=ecr_client.get_authorization_token()
+    username, password = base64.b64decode(auth_token['authorizationData'][0]['authorizationToken']).decode('utf-8').split(":")
+    # can add --verbose argument to notation verify if needed
+    result=subprocess.run(['notation', 'verify', '-u', username, '-p', password, container_image, '--verbose'])
+    return result
+
+def lambda_handler(event, context):
+
+    # only run when we first provision the container
```

Comment:
should we add a comment that if needed customers can modify this to verify signatures not just at  the initial deploy time, but also at every task launch?